games-strategy/warmux/warmux-11.04.1-r2.ebuild

```diff
@@ -52,15 +57,14 @@ src_configure() {
 		--with-localedir-name="${EPREFIX}"/usr/share/locale \
 		--with-datadir-name="${EPREFIX}"/usr/share/${PN} \
```

Comment:
Can remove these two.

Passing localedir notably tends to be a bad leftover from games.eclass because datadir=/usr/games made locales go in /usr/games/locale and the like, but no one cleaned them up in migration.

See defaults:
```
  --with-datadir-name=DIR specify where datas are installed (default:
                          ${prefix}/share/warmux)
  --with-localedir-name=DIR
                          specify where locales are installed (default:
                          ${prefix}/share/locale)
```